src/cls/lock/cls_lock_types.h

```diff
@@ -14,40 +14,40 @@
 #define LOCK_FLAG_MUST_RENEW 0x2   /* lock must already be acquired */
 
 enum ClsLockType {
-  LOCK_NONE                = 0,
-  LOCK_EXCLUSIVE           = 1,
-  LOCK_SHARED              = 2,
-  LOCK_EXCLUSIVE_EPHEMERAL = 3, /* lock object is removed @ unlock */
+  LOCK_TYPE_NONE                = 0,
```

Comment:
might want to switch to scoped enum, like
```c++
enum class ClsLockType {
  NONE = 0,
  EXCLUSIVE = 1,
  SHARED = 2,
  EXCLUSIVE_EPHEMERAL = 3, /* lock object is removed @ unlock */
};
```

Reply:
Thanks for the suggestion.